.msggen.json

```diff
@@ -696,13 +696,14 @@
             "ListPays.pays[]": 1
         },
         "ListpeersPeers": {
-            "ListPeers.peers[].channels[]": 4,
+            "ListPeers.peers[].channels[]": 5,
```

Comment:
This is bad, as it breaks backwards compatibility with any existing client. Why was this renumbering necessary? Notice that `.msggen.json` is not an artifact that needs cleaning, rather it tracks field numbers we have assigned in the past and MUST not change.

Reply:
Resetting and then regenerating the gtpc bindings should get us back on track. I.e., the new num_channels field will get a new, previously unused, field number, and all others retain their current number.

Reply:
I think CI was complaining about that if I do it otherwise. This msggen and grpc stuff was kind of a mystery to me, as my local repo did not do the automated changes, only parts of it by calling `PYTHONPATH=contrib/msggen python3 contrib/msggen/msggen/__main__.py`. But not these ones, I actually took these changes from the CI error log :D

Reply:
Lets see what CI tells us about [51d56c4](https://github.com/ElementsProject/lightning/pull/5968/commits/51d56c49e717bdc164172ef9f0d61cc0a215f716) ...